easybuild/easyconfigs/a/ABINIT/ABINIT-9.6.2-intel-2022a.eb

```diff
@@ -0,0 +1,71 @@
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = '9.6.2'
+
+homepage = 'https://www.abinit.org/'
+description = """
+ABINIT is a package whose main program allows one to find the total energy, charge density and electronic structure of
+systems made of electrons and nuclei (molecules and periodic solids) within Density Functional Theory (DFT), using
+pseudopotentials and a planewave or wavelet basis.
+"""
+
+toolchain = {'name': 'intel', 'version': '2022a'}
+toolchainopts = {'usempi': True, 'openmp': True, 'pic': True}
+
+source_urls = ['https://www.abinit.org/sites/default/files/packages/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['b018c2ff24338a5952c5550a7e09d4c7793b62402c7aa4e09273e9a666e674fb']
+
+builddependencies = [
+    ('Python', '3.10.4'),
+]
+dependencies = [
+    ('libxc', '5.2.3'),
+    ('netCDF', '4.9.0'),
+    ('netCDF-Fortran', '4.6.0'),
+    ('HDF5', '1.12.2'),
+    ('Wannier90', '3.1.0'),
+]
+
+# Ensure MPI with intel wrappers.
+configopts = '--with-mpi="yes" '
+configopts += ' FC="mpiifort" CC="mpiicc" CXX="mpiicpc" '
+
+# Enable OpenMP
+configopts += '--enable-openmp="yes" '
+
+# BLAS/Lapack from MKL
+configopts += '--with-linalg-flavor=mkl '
+
+# FFTW from MKL
+configopts += '--with-fft-flavor=dfti '
+
+# libxc support
+configopts += '--with-libxc=${EBROOTLIBXC} '
+
+# hdf5/netcdf4 support
+configopts += '--with-netcdf="${EBROOTNETCDF}" '
+configopts += '--with-netcdf-fortran="${EBROOTNETCDFMINFORTRAN}" '
+configopts += '--with-hdf5="${EBROOTHDF5}" '
+
+# Wannier90
+configopts += '--with-wannier90="${EBROOTWANNIER90}" '
+preconfigopts = 'export WANNIER90_LIBS="-L$EBROOTWANNIER90/lib -lwannier" && '
+
+# Enable double precision for GW calculations
+configopts += '--enable-gw-dpc '
+
+# Enable OpenMP
+configopts += '--enable-openmp '
+
```

Comment:
```suggestion
```

No longer needed.

Reply:
Thanks! Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/16674/commits/258ef7125f58c3e19be2a2777d798cebcc5448f9